src/pair.py

```diff
@@ -3,3 +3,25 @@
 # Part of azplugins, released under the BSD 3-Clause License.
 
 """Pair potentials."""
+
+from hoomd.azplugins import _azplugins
+from hoomd.data.parameterdicts import TypeParameterDict
+from hoomd.data.typeparam import TypeParameter
+from hoomd.md import pair
+
+
+class Hertz(pair.Pair):
+    """Hertz potential."""
```

Comment:
Please begin to copy over some class documentation from old code! Check in with me about what to keep vs. remove.